### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by dfsalam**
I´m adding the index.html, styles.css and the lintern files.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+h1 {
```
Comment (nedjwab) [inline styles.css:1]: [ ] kindly use a CSS class for your header. it's one of the HTML and CSS best practices and it's on the project requirements. 👇
css

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @dfsalam 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
`[ ] Pull Request should have a descriptive title telling you what changes are inside.(example: Setup project ). kindly fix that 🤓. `

` [ ]  Pull Request should have a summary describing introduced changes in more detail. (example: add HTML file, CSS classes...).kindly fix that 🤓.`

`[ ] kindly  Add a .gitignore file that includes a test.MD file and node modules .it's on the project requirements. 👇`

gitignore
testignore

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css (hunk 1 above)
Comment (dfsalam) [reply styles.css:1]: Thank you for your guidance @nedjwab 🥇 

I follow your instructions and change the pull request title and descripction. Also I add the .gitignore file to the repository.

I await any further correction.

Comment (dfsalam) [reply styles.css]: I already did a new commit with the style correction adding the class to the h1 tag.

Comment (dfsalam) [reply styles.css]: Hello @nedjwab 
I would like to know if you check the changes in my project. 😰

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @dfsalam , 

I am Favour 👩🏽 , your reviewer for this project milestone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 
- GitHib flow used ✔️ 
- Node modules and test.md files ignored ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**